paparazzi/src/main/java/app/cash/paparazzi/PaparazziSdk.kt

```diff
@@ -362,6 +361,12 @@ public class PaparazziSdk @JvmOverloads constructor(
         RenderAction.getCurrentContext().sessionInteractiveData.choreographerCallbacks
       choreographerCallbacks.execute(currentTimeNanos, Bridge.getLog())
 
+      // The choreographer needs to be manually ticked in order for the frame time to become visible to the native layer
+      // which is necessary in order for ripples to work.
+      if (currentTimeNanos != 0L) {
+        Choreographer_Delegate.doFrame(currentTimeNanos)
```

Comment:
This doesn't feel great but I haven't found a lighter weight option that prevents the crash. I also tried removing the callback invoking code above assuming that this might re-invoke the callbacks but it doesn't :shrug:

The tests are passing.